ouroboros-consensus-cardano/tools/db-analyser/Main.hs

```diff
@@ -149,6 +150,11 @@ parseAnalysis = asum [
           long "count-blocks"
         , help "Count number of blocks processed"
         ]
+    , checkNoThunksParser
+    , flag' TraceLedgerProcessing $ mconcat [
+          long "trace-lgr"
```

Comment:
Please rename to `trace-ledger`. Please also enrich the `help` message so that it indicate the following two things to some degree.

- These phases are determined by logic specific to this tool.
- They are emitted via the GHC `eventlog`.